config/config.js

```diff
@@ -24,12 +24,12 @@ module.exports = {
    */
   description:
     'ServiceNow Security Incident Response (SIR) allows your organization to manage the life cycle of your security incidents from initial analysis to containment, eradication, and recovery.',
-  entityTypes: ['ipv4', 'email', 'domain', 'hash', 'cve'],
+  entityTypes: ['IPv4', 'email', 'domain', 'hash', 'cve'],
```

Comment:
I think for consisteny we should also update the `config.js` to use the same entity types:
```
"entityTypes": ["IPv4", "email", "domain", "MD5", "SHA1", "SHA256", "cve"]
```